configure

```diff
@@ -1039,7 +1040,12 @@ get_compiler_version()
 	# The last part ({ read first rest ; echo $first ; }) is a workaround
 	# to OS X's egrep only returning the first match.
 	cc_vendor=$(echo "${vendor_string}" | egrep -o 'icc|gcc|clang|emcc|pnacl|IBM' | { read first rest ; echo $first ; })
-	cc_version=$(echo "${vendor_string}" | egrep -o '[0-9]+\.[0-9]+\.?[0-9]*' | { read first rest ; echo ${first} ; })
+    if [ "$cc_vendor" == "gcc" -o "$cc_version" == "clang" ]
```

Comment:
`$cc_version` -> `$cc_vendor`. You could also add icc since it supports `-dumpversion` too.

Reply:
Also, only `=` is POSIX sh, not `==`.